### ISSUE TITLE
Install method on linux should use git clone

### ISSUE THREAD

**Issue by ArrODevv** [flutter/website#11498]
### Page URL

https://docs.flutter.dev/get-started/install/linux/android#download-then-install-flutter

### Page source

https://github.com/flutter/website/tree/main/src/content/get-started/install/linux/android.md

### Describe the problem

On the page listed above, there is a download link for a `.tar.xz` and every time (2 times to be precise) I install flutter from this file, `flutter doctor` complains that the directory should be a git clone of [flutter/flutter](https://github.com/flutter/flutter). After cloning said repo it immediately worked. It's not a fatal issue, but it does decrease user comfort.

Sorry for that wall of text, let me know if you need more infos or if this is a duplicate (I couldn't find an issue for this) or really anything.

Thank you!

### Expected fix

I think the first step shouldn't be downloading the `tar.xz` archive, but rather opening a terminal and running `git clone [...]`.

### Additional context

I use Firefox 133.0.3 and am following the installation tutorial for flutter android app development.

### I would like to fix this problem.

- [X] I will try and fix this problem on docs.flutter.dev.

**Comment by sfshaza2** [comment]
@antfitch, would you mind taking a look at this?

**Comment by antfitch** [comment]
Absolutely.

**Comment by antfitch** [comment]
My hunch is that a developer should not have to clone flutter/flutter to build a Flutter Android app. I suspect that a developer should clone this repository only if they feel like making their own changes to the Flutter project or taking a sneak peak at upcoming features.

@craiglabenz is this something you can chime in on? @ArrODevv when you were getting those error messages, had you ever cloned flutter/flutter on the machine you were using before?

**Comment by ArrODevv** [comment]
No, it was my first time installing flutter. I did everything according to the guide. At first I thought there was a good reason it needed a git clone?

A bit after posting this issue, I also installed flutter on my windows machine and there was a git clone in the zip file.

Thanks for your time so far!

**Comment by antfitch** [comment]
@khanhnwin sending this one your way for triaging. Has @ArrODevv uncovered an installation issue? In summary, should a developer have to clone flutter/flutter to build a Flutter Android app?

**Comment by domesticmouse** [comment]
The download zip file for macOS also includes a git repository, and it is included to enable the developer to change Flutter version, which happens every couple of weeks or so. So yeah, I'd say that a git clone, either via the git tool directly, or by downloading a fully populated git tree (which is what the downloads are supposed to be) is required.

If the linux download build workflow is not properly populating the git tree in the download file, then we have an issue with the workflow for constructing the linux download.

**Comment by antfitch** [comment]
Okay great! I'll add a note in the docs about what to expect with `.tar.xz`.

**Comment by antfitch** [comment]
Tentatively closing because I think this was an issue with a specific `.tar.xz` file. Glad we were able to capture this through the docs steps not working as expected.
